### DISCUSSION TITLE
Any reason not to document `$event`?

### DISCUSSION THREAD

**Discussion by asbjornu** [alpinejs/alpine#4172] [4. General]
The magic `$event` object isn't documented on [alpinejs.dev](https://alpinejs.dev/). Is it omitted for a particular reason, or would you accept a PR adding it to the documentation? Are there other magic objects that are undocumented?

**Comment by SimoTod** [comment]
It's documented [here](https://alpinejs.dev/directives/on#the-event-object) as it's not broadly available but it's just a thing you can use with x-on

**Reply by asbjornu** [reply]
Right, that makes sense. Thanks!
